### ISSUE TITLE
NTR: Multiple organ system failure

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#5811]
**Preferred term label:**
Multiple organ system failure

**DEF**: altered function in two or more organ systems during an acute illness such that homeostasis cannot be maintained without intervention
SOURCE: https://litfl.com/multiple-organ-dysfunction-syndrome/

**SYN**: multisystem organ failure
**SYN**: multi-organ failure
**SYN**: multiple system organ failure
**SYN**: multiple organ system failure

**Parent term (use hpo.jax.org/app)**
'Constitutional symptom' ?

See: https://github.com/obophenotype/mammalian-phenotype-ontology/issues/3207

**Comment by pnrobinson** [comment]
I think that this is better conceptualized as a disease called 'Multiple Organ Dysfunction Syndrome'
This is way too bundled and variable to be a good HPO term.
The following has a good description that could be used to write  definition for MONDO
https://www.ncbi.nlm.nih.gov/books/NBK6868/

**Comment by pnrobinson** [comment]
see https://github.com/monarch-initiative/mondo/issues/1822
